### PR TITLE
Add AES SIV ciphers to default provider

### PR DESCRIPTION
**Pull request by slontis**
##### Checklist

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: test/recipes/30-test_evp.t [hunk 1]
```diff
@@ -33,11 +33,13 @@ push @configs, 'fips.cnf' unless $no_fips;
 
 my @files = qw( evpciph.txt evpdigest.txt );
 my @defltfiles = qw( evpencod.txt evpkdf.txt evppkey_kdf.txt evpmac.txt
-    evppbe.txt evppkey.txt evppkey_ecc.txt evpcase.txt evpaessiv.txt
-    evpccmcavs.txt );
+    evppbe.txt evppkey.txt evppkey_ecc.txt evpcase.txt evpccmcavs.txt );
 my @ideafiles = qw( evpciph_idea.txt );
 push @defltfiles, @ideafiles unless disabled("idea");
 
+my @sivfiles = qw( evpaessiv.txt );
```
Comment (levitte) [inline test/recipes/30-test_evp.t]: If you have this file already, might as well make sure it's here

**Comment by levitte** [comment]
Except for the lack of test data, this looks fine. Not sure why this depends on chacha...

File: test/recipes/30-test_evp.t (hunk 1 above)
Comment (slontis) [reply test/recipes/30-test_evp.t]: The file that already exists..

**Comment by slontis** [comment]
> Not sure why this depends on chacha...

Each cipher PR checked in means a rebase of any other cipher related PR's.
evp_enc.c, provider_algs.h, defltprov.c always need rebasing..

**Comment by slontis** [comment]
rebased and separated cipher low level implementation from dispatch

**Comment by slontis** [comment]
Moved out of WIP: and rebased.

**Comment by slontis** [comment]
ping

**Comment by slontis** [comment]
Thanks. Merged to master..

**Pull request closed without merging**